### PR TITLE
clean up getdomain logic

### PR DESCRIPTION
**Pull request by jreadey**
Clean up get domain logic.

### CODE REVIEW

File: hsds/servicenode_lib.py [hunk 1]
```diff
@@ -63,7 +64,33 @@ async def getDomainJson(app, domain, reload=False):
...
+    try:
+        domain_json = await http_get(app, req, params=params)
+    except HTTPNotFound:
+        log.warn(f"domain: {domain} not found")
+        raise
+    except HTTPGone:
+        log.warn(f"domain: {domain} has been removed")
+        raise
+    except ClientResponseError as ce:
+        # shouldn't get this if we are catching relevent exceptions
```
Comment (mattjala) [inline hsds/servicenode_lib.py:76]: `relevent` -> `relevant`

File: hsds/util/domainUtil.py [hunk 2]
```diff
@@ -268,7 +268,7 @@ def validateDomainKey(domain_key):
 
 
 def getDomainFromRequest(request, validate=True):
-    # print("gotDomainFromRequest:", request, "validate=", validate)
+    print(f"tmp - getDomainFromRequest: {request}, validate={validate}")
```
Comment (mattjala) [inline hsds/util/domainUtil.py:271]: Looks like a debug print was left in

File: hsds/util/domainUtil.py [hunk 3]
```diff
@@ -279,7 +279,7 @@ def getDomainFromRequest(request, validate=True):
         domain = request.headers["X-Hdf-domain"]
     else:
         return None
-
+    print(f"tmp - domain: {domain}")
```
Comment (mattjala) [inline hsds/util/domainUtil.py:282]: Same here

File: hsds/servicenode_lib.py (hunk 1 above)
Comment (jreadey) [reply hsds/servicenode_lib.py]: fixed

File: hsds/util/domainUtil.py (hunk 2 above)
Comment (jreadey) [reply hsds/util/domainUtil.py]: fixed

File: hsds/util/domainUtil.py (hunk 3 above)
Comment (jreadey) [reply hsds/util/domainUtil.py]: fixed

**Pull request merged**